javascript/javascript3/week3/Promises/index.js

```diff
@@ -0,0 +1,36 @@
+const div = document.createElement("div");
+const urls = [
+  'https://api.github.com/search/repositories?q=user:Aswini-D3',
+  'https://api.github.com/search/repositories?q=user:STEP0HEN',
+  'https://api.github.com/search/repositories?q=user:AkoAyukngat'
+];
```

Comment:
This is not reusable: what if you'd need URLs for many users, not just 3? 
you could do 
`const url = 'https://api.github.com/search/repositories?q=user:`, and then add usernames